src/components/DisciplineListItem.tsx

```diff
@@ -0,0 +1,24 @@
+import React, { ReactElement } from 'react'
+
+import { Discipline } from '../constants/endpoints'
+import { childrenDescription, childrenLabel } from '../services/helpers'
+import ListItem from './ListItem'
+
+interface DisciplineListItemProps {
+  item: Discipline
+  onPress: () => void
+  badge: boolean
+}
+
+const DisciplineListItem = ({ item, onPress, badge }: DisciplineListItemProps): ReactElement | null => {
+  const { numberOfChildren, title, icon } = item
+  const badgeLabel = badge ? numberOfChildren.toString() : undefined
+  const description = badge ? childrenLabel(item) : childrenDescription(item)
+
+  if (numberOfChildren === 0) {
+    return null
+  }
```

Comment:
So this means if a discipline does not have any Subdiciplines the Item will not render?

Reply:
Yes. Was the same way before with the only difference that this was handled elsewhere (using filter e.g. in HomeScreen)